File: app/src/main/java/ru/androidacademy/ateam/ui/activity/MainActivity.java
package ru.androidacademy.ateam.ui.activity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;
import androidx.appcompat.app.AppCompatActivity;
import ru.androidacademy.ateam.R;
import ru.androidacademy.ateam.ui.activity.addword.ListWordActivity;

public class MainActivity extends AppCompatActivity {

    TextView mGameName;
    Button mButtonStart;
    Button mButtonContinue;
    Button mButtonRules;
    Button mButtonHelp;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);



        mButtonStart = findViewById(R.id.start_new_button);
        mButtonRules = findViewById(R.id.rules_button);
        mButtonContinue = findViewById(R.id.continue_button);

        mButtonStart.setOnClickListener(new View.OnClickListener(){
            public void onClick(View v){
                Intent intent = new Intent(MainActivity.this, SettingsActivity.class);
                startActivity(intent);
            }
        });

        mButtonRules.setOnClickListener(new View.OnClickListener(){
            public void onClick(View v){
                Intent intent = new Intent(MainActivity.this, RulesActivity.class);
                startActivity(intent);
            }
        });

        mButtonContinue.setOnClickListener(new View.OnClickListener(){
            public void onClick(View v){
                Intent intent = new Intent(MainActivity.this, ListWordActivity.class);
                startActivity(intent);
            }
        });
    }





}

File: app/src/main/java/ru/androidacademy/ateam/ui/activity/MainActivity.kt
package ru.androidacademy.ateam.ui.activity

import android.content.Intent
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*
import ru.androidacademy.ateam.R


class MainActivity : MvpAppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setListeners()

    }

    private fun setListeners() {
        activity_main_button_new_game.setOnClickListener {
            startNewGame()
        }
    }

    private fun startNewGame() {
        val intent = Intent(this, SettingsActivity::class.java)
        startActivity(intent)
    }
}